src/app/app.component.html

```diff
@@ -1,3 +1,3 @@
 <ion-app>
-  <ion-router-outlet></ion-router-outlet>
+  <ion-router-outlet swipeGesture="false" animated="false"></ion-router-outlet>
```

Comment:
Can you explain why this change was made?

Reply:
Again, this change was introduced by @shreedhanu08

Reply:
Interesting, where was it introduced? It not in the main branch at this time:

https://github.com/trustthevote/MVP-App-TestHarness/blob/main/src/app/app.component.html

Reply:
This commit: e7b7a6c330f597ef8dc2c2412abf8f2f6d62ba71

I used a branch where you had created a `PairingCodeScreen`

Reply:
Let's follow up here: https://github.com/trustthevote/MVP-App-TestHarness/pull/48#discussion_r846097553

Reply:
Resolution summarized here: https://github.com/trustthevote/MVP-App-TestHarness/pull/48#issuecomment-1093634620